services/Skype.json

```
@@ -7,7 +7,7 @@
         },
         "privacy": {
             "location": "https://privacy.microsoft.com/en-gb/privacystatement",
-            "contentSelector": "h1, #psp_last_updated, span.Description, .div_content",
+            "contentSelector": "h1, #psp_last_updated, span.Description:nth-child(1), .div_content",
```

More idiomatic 🙂 
```suggestion
            "contentSelector": "h1, #psp_last_updated, span.Description:first-child, .div_content",
```

---

Actually after looking into this selector and the structure of the page more in depth, it turns out what we really want is to include the `.Description` only for the `.header_content`, which turns out to also be the only one to be a `:first-child` of its parent, but it is a coincidence 🙂

So what we want is:
```suggestion
            "contentSelector": "h1, #psp_last_updated, .header_content .Description, .div_content",
```